assets/chezmoi.io/docs/user-guide/templating.md

```diff
@@ -93,22 +93,33 @@ $ chezmoi edit --apply ~/.zshrc
 
 ## Testing templates
 
-Templates can be tested with the `chezmoi execute-template` command which treats
-each of its arguments as a template and executes it. This can be useful for
-testing small fragments of templates, for example:
+Templates can be tested and debugged with the `chezmoi execute-template`
+command which treats each of its arguments as a template and executes it.
+It will then interpret the templates and output the result to standard
+output.
+
+This can be useful for testing small fragments of templates, for example:
 
 ```console
 $ chezmoi execute-template '{{ .chezmoi.hostname }}'
 ```
 
 If there are no arguments, `chezmoi execute-template` will read the template
-from the standard input. This can be useful for testing whole files, for example:
+from the standard input. This can be useful for testing whole files,
+for example:
```

Comment:
```markdown
Without arguments, `chezmoi execute-template` will read the template from standard input,
which is useful for testing whole files:
```

Reply:
I’d like to eliminate the `for example`:, which is part of why I suggested the rewording.